respx/router.py

```diff
@@ -154,6 +154,14 @@ def request(
         name: Optional[str] = None,
         **lookups: Any,
     ) -> Route:
+        if lookups:
+            # Validate that lookups doesn't contain method or url
+            pattern_keys = {p.split("__")[0] for p in lookups.keys()}
```

Comment:
Think splitting on first `__` is enough here.

```suggestion
            pattern_keys = {p.split("__", 1)[0] for p in lookups.keys()}
```